src/services/review/RepoCheckerService.ts

```diff
@@ -819,6 +852,29 @@ export default class RepoCheckerService {
     })
   }
 
+  permalinkNormaliser(permalink: string) {
+    let finalPermalink = permalink
+
+    if (finalPermalink.startsWith(`'`)) {
+      finalPermalink = finalPermalink.slice(1)
+    }
+    if (finalPermalink.endsWith(`'`)) {
+      finalPermalink = finalPermalink.slice(0, -1)
+    }
+    if (finalPermalink.startsWith(`"`)) {
+      finalPermalink = finalPermalink.slice(1)
+    }
+    if (finalPermalink.endsWith(`"`)) {
+      finalPermalink = finalPermalink.slice(0, -1)
+    }
```

Comment:
just thinking out loud here - would it be possible for a permalink to have a starting `'` but not a terminating `'`? (also for double quotes) 

this, for example, would also sanitise `'"some_permalink"'` to `some_permalink` but not `"'some_permalink'"` (only removes outer `"`) - again, this is contrived but i'm not sure what behaviour we're aiming for here

Reply:
using regex now [6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85)